webapp/src/data.ts

```diff
@@ -275,6 +275,7 @@ export function stripProtocol(path: string) {
 }
 
 export function invalidate(prefix: string) {
+    prefix = prefix.replace(/:\*$/, ':'); // remove trailing "*";
```

Comment:
all headers:* were wrong

Reply:
oh wow!

Reply:
Yup, we didn’t invalidate tons of stuff ?!??!